governance/polls/Vault Compensation Working Group Payment - January 11, 2021.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Vault Compensation Working Group Payment - January 11, 2021
+summary: Signal your support or opposition to paying 12,700 Dai from the surplus buffer to the vault compensation working group for past labor.
+discussion_link: https://forum.makerdao.com/t/signal-request-vault-compensation-working-group-payment/5562
+vote_type: Plurality Voting
+options:
```

Comment:
We have categories now:
`categories:`
`   - Other`
See Inclusion Poll - $mip_code - $date_MONTH_DD,_YYYY.md template for syntax if you have trouble. Feel free to ignore this and I'll fix it later.

Reply:
thanks